instrumentation/kafka/kafka-clients/kafka-clients-common/library/src/main/java/io/opentelemetry/instrumentation/kafka/internal/KafkaMetricRegistry.java

```diff
@@ -141,5 +141,9 @@ private static AutoCloseable createObservable(
     throw new IllegalStateException("Unrecognized instrument type. This is a bug.");
   }
 
+  private static double value(KafkaMetric kafkaMetric) {
+    return kafkaMetric.measurable().measure(kafkaMetric.config(), System.currentTimeMillis());
```

Comment:
Wouldn't `metricValue()` be a bit more generic choice? (And it does not require passing time, it uses internal kafka clock)

Reply:
`metricValue` is not available in `kafka-clients:0.11.0.0`. The time abstraction in kafka is probably to ease faking time in unit tests, the only real implementation is `SystemTime` that also calls `System.currentTimeMillis()`.